modules/ml/include/opencv2/ml.hpp

```diff
@@ -104,7 +104,7 @@ enum SampleTypes
 It is used for optimizing statmodel accuracy by varying model parameters, the accuracy estimate
 being computed by cross-validation.
  */
-class CV_EXPORTS ParamGrid
+class CV_EXPORTS_W ParamGrid
```

Comment:
`CV_PROP_RW` can be also added before `minVal` and `maxVal` to make them visible in python.